lab-adan/test/server-test.js

```diff
@@ -0,0 +1,95 @@
+'use strict';
+
+const superagent = require('superagent');
+const expect = require('expect');
+const server = require('../lib/server.js');
+const GameScore = require('../model/gamescore.js');
+const PORT = process.env.PORT || 3000;
+
+describe('GameScore Constructor', () => {
+  it('Should return a GameScore Object', () => {
+    let newGame = new GameScore(1, 'tetris', 9000);
+    expect(newGame.id).toEqual(1);
+    expect(newGame.name).toEqual('tetris');
+    expect(newGame.score).toEqual(9000);
+  });
+});
+
+describe('/api/gamescore routes', () => {
+  let testGame;
+  before(done => {
+    server.listen(PORT, () => done());
+  });
+  after(done => {
+    server.close(() => done());
+  });
+  describe('POST', () => {
+    it('Should respond 201 with stringified JSON', done => {
+      superagent.post(`localhost:${PORT}/api/gamescore`)
+        .send(JSON.stringify({name: 'Sonic', score: '45000'}))
+        .end((err, res) => {
+          if (err) return done(err);
+          expect(res.status).toEqual(201);
+          expect(res.body.id).toExist();
```

Comment:
These tests are failing because of the way that you set up your post route handler. Throw in a `console.log(res.body)` here and you'll see what I mean.